alienfile

```diff
@@ -62,6 +62,9 @@ my %bad_versions = map { $_ => 1 } (
   '2.6.0','2.6.1','2.6.2','2.6.3','2.6.4','2.6.5','2.6.6','2.6.7','2.6.8','2.6.9','2.6.10','2.6.11','2.6.12','2.6.13','2.6.14','2.6.15','2.6.19','2.6.20','2.6.25',
   '2.7.0','2.7.1',
   '2.9.4',
+  (map {'2.11.' . $_} (0 .. 9)),
+  (map {'2.12.' . $_} (0 .. 9)),
+  (map {'2.13.' . $_} (0 .. 6)),
```

Comment:
Can you add a comment as to the reason?  ie, the CVE (s?). I know that we haven't done this previously but seems like useful information.

Reply:
Done.  Let me know if you want me to adapt the wording and/or position.